app/components/Settings/Settings.react.js

```diff
@@ -300,222 +297,141 @@ class Settings extends Component {
+                                        </div>
+                                    ) : (
+                                        <div>
+                                            <p>
+                                                {`Plotly is automatically initializing a
+                                                  unique SSL certificate and URL for you.
+                                                  This can take up to 10 minutes.`}
+                                            </p>
+                                            <p>
+                                                {`It has been ${timeElapsed}`}.
+                                            </p>
+                                            <p>
+                                                {`Once this is complete, you'll be able to
+                                                  query your databases from `}
+                                                <Link href={`${plotlyUrl}/create?upload=sql`}>
+                                                        Plotly
+                                                </Link>
+                                                {'.'}
+                                            </p>
+                                        </div>
+                                    )}
                                 </div>
                             ) : (
+                                <p>You must connect to a data store first.</p>
+                            )}
+                        </TabPanel>
+
+                        <TabPanel>
+                            {this.props.connectRequest.status === 200 ? (
                                 <div>
-                                    {`Before you can query, wait until Plotly has
-                                      finished Step 3 for you.`}
+                                    {httpsServerIsOK ? (
+                                        <div id="test-ssl-initialized">
+                                            <p>
+                                                You're now ready to explore this datastore on Plotly!
+                                                Plotly has generated a local URL for this app
+                                                through which it will securely send queries: 
+                                            </p>
+                                            <div style={{textAlign: 'center'}}>
+                                                <strong><code>{connectorUrl}</code></strong>
+                                            </div>
+                                            <p>
+                                                This URL is local - no one can access it except this computer or server.
+                                                Click below to connect and start writing queries:
+                                            </p>
+                                            <a 
```

Comment:
Instead of `<a/>` we use `<Link/>`. `<a/>` will open the window in a new electron tab rather than the web browser. `<Link/>` will open the link the actual web browser.